frida-gum-sys/build.rs

```diff
@@ -242,6 +242,14 @@ fn main() {
     #[cfg(all(feature = "js", target_os = "linux"))]
     println!("cargo:rustc-link-lib=dylib=stdc++");
 
-    #[cfg(all(feature = "js", target_os = "macos"))]
-    println!("cargo:rustc-link-lib=resolv");
+    // #[cfg(all(feature = "js", target_os = "macos"))]
+    // println!("cargo:rustc-link-lib=resolv");
+    #[cfg(all(feature = "js",target_os = "macos", target_arch = "aarch64"))]
+    {
+    println!("cargo:rustc-link-search=native={}", "$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/darwin-x86_64/sysroot/");
```

Comment:
We should detect if we are on darwin or linux, and use the correct prebuilt.

Also, would be a good idea to test to see if `ANDROID_NDK_HOME` is defined and print an error message if not.